contrib/win32/openssh/keygen.vcxproj

```diff
@@ -195,6 +195,9 @@
       <EntryPointSymbol>wmainCRTStartup</EntryPointSymbol>
       <AdditionalOptions>/debug /debugtype:cv,fixup /opt:ref /opt:icf /incremental:no %(AdditionalOptions)</AdditionalOptions>
     </Link>
+    <Manifest>
```

Comment:
why is this change no there for ARM64 platform?

Reply:
looks like all the proj files here don't have this change for ARM64 platform.

Reply:
I will look at this.

Reply:
I will take care of this.